config/.env.backup.20250910_110945

```
@@ -0,0 +1,239 @@
+############
+# [required] 
+# n8n credentials - use the command `openssl rand -hex 32` to generate both
+#   openssl is available by default on Linux/Mac
+#   For Windows, you can use the 'Git Bash' terminal installed with git
+#   Or run the command: python -c "import secrets; print(secrets.token_hex(32))"
+############
+
+N8N_ENCRYPTION_KEY=@mb29_aENnhg>-%P%*64]2vE>eU$j^o#
+N8N_USER_MANAGEMENT_JWT_SECRET=P4VF21=Uzt,zL^2CboXsOri%Bi|&|p:A
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)